src/containers/Dashboard.vue

```diff
@@ -388,76 +387,8 @@ export default Vue.extend({
       return arr && arr.length !== 0 && arr[0] !== '' ? arr : ['N/A'];
     },
 
-    getCourseCodesArray(): readonly CourseTaken[] {
-      const courses: CourseTaken[] = [];
-      this.semesters.forEach(semester => {
-        semester.courses.forEach(course => {
-          const [subject, number] = course.code.split(' ');
-          courses.push({
-            code: `${course.lastRoster}: ${subject} ${number}`,
-            subject,
-            courseId: course.crseId,
-            number,
-            credits: course.credits,
-            roster: course.lastRoster,
-          });
-        });
-      });
-      courses.push(...getCourseEquivalentsFromUserExams(this.onboardingData));
-      return courses;
-    },
-
-    getRequirementTypeDisplayName(type: string): string {
-      return type.charAt(0).toUpperCase() + type.substring(1);
-    },
-
     recomputeRequirements(): void {
-      const groups = computeRequirements(
-        this.getCourseCodesArray(),
-        this.onboardingData.college,
-        this.onboardingData.major,
-        this.onboardingData.minor
-      );
-      // Turn result into data readable by requirements menu
-      const singleMenuRequirements = groups.map(group => {
-        const singleMenuRequirement: SingleMenuRequirement = {
-          ongoing: [],
-          completed: [],
-          name: `${
-            group.groupName.charAt(0) + group.groupName.substring(1).toLowerCase()
-          } Requirements`,
-          group: group.groupName.toUpperCase() as 'COLLEGE' | 'MAJOR' | 'MINOR',
```

Comment:
The ALL-CAPS is only used for getting color from `reqGroupColorMap`, so I changed `reqGroupColorMap` to make the fields be `College | Major | Minor` instead of `COLLEGE | MAJOR | MINOR`